client/my-sites/people/team-list/test/team.jsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'Team', function() {
+	let Card,
+		PeopleListSectionHeader,
+		InfiniteList,
+		Team,
+		configMock,
+		deterministicStringify;
+
+	useFakeDom();
+	useMockery( mockery => {
+		configMock = sinon.stub();
+		configMock.isEnabled = sinon.stub();
+		mockery.registerMock( 'config', configMock );
+
+		deterministicStringify = sinon.stub();
+		mockery.registerMock( 'lib/deterministic-stringify', deterministicStringify );
+
+		mockery.registerMock( 'lib/sites-list', () => null );
+	} );
+
+	before( function() {
+		Card = require( 'components/card' );
+		PeopleListSectionHeader = require( 'my-sites/people/people-list-section-header' ).default;
+		InfiniteList = require( 'components/infinite-list' );
+
+		Team = require( '../team' ).Team;
+	} );
+
+	it( 'renders infinite list when site is specified and any users have been loaded', function() {
+		deterministicStringify
+			.withArgs( { option1: 'option1value', option2: 'option2value' } )
+			.returns( 'list key' );
+
+		const wrapper = shallow(
+			<Team
+				fetchInitialized={ true }
+				fetchingUsers={ false }
+				fetchOptions={ { option1: 'option1value', option2: 'option2value', number: 10, offset: 2 } }
+				site={ 'site1' }
+				users={ [ 'user1' ] }
+				excludedUsers={ [] }
+			/>
+		);
+
+		expect( wrapper.find( Card ).find( InfiniteList ).key() ).to.equal( 'list key' );
+	} );
+
+	it( 'renders translated team header text by default', function() {
+		const translate = sinon.stub();
+		translate
+			.withArgs( 'Team' )
+			.returns( 'team label' );
+
+		const wrapper = shallow(
+			<Team
+				fetchInitialized={ true }
+				fetchingUsers={ false }
+				fetchOptions={ {} }
+				users={ [] }
+				excludedUsers={ [] }
+				translate={ translate }
+			/>
+		);
+
+		expect( translate.callCount ).to.equal( 1 );
+
+		expect( wrapper.find( PeopleListSectionHeader ).prop( 'label' ) )
+		.to.equal( 'team label' );
+	} );
+
+	it( 'renders translated header text for search when there are search results', function() {
+		const translate = sinon.stub();
```

Comment:
I'm not sure why a spy would be necessarily simpler - can you give me an example?

The code coverage from these tests are clearly quite minimal - I want to get the infrastructure in place so that adding tests is easier.